### ISSUE TITLE
random bug i found with genetics that iv no idea how it happend

### ISSUE THREAD

**Issue by Syilumi** [Bubberstation/Bubberstation#2653]
### Client Version:

515.1647

### Issue Summary:

was messing with CRISPRS had one that could only infect through blood,
was planning on replacing monkified with telekenesis

gave a monkey the virus got a crispr injector was solving the headache inducing gene A's and T's sometimes g's

made myself a monkey so i could replace it, realised monkies cant use genetic consoles so tried to take mutadone to unmonkey

didnt work gave me the crispr virus somehow and forced me to just round remove via cryo cause admins couldnt even fix it

### Round ID:

41

### Reproduction Steps:

i have no idea where the issue started or ended

**Comment by JagOfTroy** [comment]
To restate this so I understand correctly, you tried to use mutadone to unmonkey yourself but instead, it gave you the crispr virus that couldn't be removed even by the admin panel?

**Comment by Syilumi** [comment]
> To restate this so I understand correctly, you tried to use mutadone to unmonkey yourself but instead, it gave you the crispr virus that couldn't be removed even by the admin panel?

yep they even tried using the VV panel thing its how they found out it even had the monkey mutation in it

**Comment by JagOfTroy** [comment]
> > To restate this so I understand correctly, you tried to use mutadone to unmonkey yourself but instead, it gave you the crispr virus that couldn't be removed even by the admin panel?
> 
> yep they even tried using the VV panel thing its how they found out it even had the monkey mutation in it

I'll try to see if I can reproduce the results and go from there, thanks for clarifying.

**Comment by Shadow-Quill** [comment]
Feel free to re-open this if it can be reproduced.
